iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -95,7 +95,7 @@ def device_info(self, device_id):
     @return_type("bool")
     def check_time(self, max_slop = 300):
         """ Check if current system time is consistent with iotile.cloud time"""
-        cloud_time = requests.get('https://iotile.cloud').headers.get('Date', None)
+        cloud_time = requests.get('https://iotile.cloud/api/v1/server/').json()['now']
```

Comment:
I believe you need to use `.get('now', None) here otherwise you will get a `KeyError` if now doesn't exist so your `cloud_time is None` won't catch what you expect.

Reply:
Right, I forgot, will change that now